### PR TITLE
various improvements

### PR DESCRIPTION
**Pull request by jasonballinger**
please accept it fixes your website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +35,8 @@ <h1 style="color:White;font-family:Courier New;"> Welcome To Amanda's Website Ɛ
     <audio controls>
       <source src="audio/MGK - Forget Me Too.ogg" type="audio/ogg">
       <source src="audio/MGK - Forget Me Too.mp3" type="audio/mpeg">
+    </audio>
```
Comment (jasonballinger) [inline index.html:38]: i added a closing tag because you didnt have it before

File: index.html [hunk 2]
```diff
@@ -34,6 +35,8 @@ <h1 style="color:White;font-family:Courier New;"> Welcome To Amanda's Website Ɛ
...
+    <br>
```
Comment (jasonballinger) [inline index.html:39]: html best practice

File: index.html [hunk 3]
```diff
@@ -19,6 +19,7 @@
     <hr size="7" width="100%" color="#0000FF">  
     <hr size="7" width="100%" color="#9400D3">   
     <h1 style="color:White;font-family:Courier New;"> Welcome To Amanda's Website Ɛ></font> </h1>
+    <img alt="bruh" src="https://c.tenor.com/Ku-j2duk5f0AAAAC/sus.gif">
```
Comment (jasonballinger) [inline index.html:22]: dont click just trust me

**Comment by amandan55896** [comment]
lmao ok

**Pull request merged**